src/Compilers/VisualBasic/Portable/Syntax/InternalSyntax/SyntaxNodeExtensions.vb

```diff
@@ -445,10 +445,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         ' as many consecutive tokens as possible into a SkippedTokens trivia node.
         Private Class SkippedTriviaBuilder
             ' Maintain the list of trivia that we're accumulating.
-            Private _triviaListBuilder As SyntaxListBuilder(Of GreenNode) = SyntaxListBuilder(Of GreenNode).Create()
+            Private ReadOnly _triviaListBuilder As SyntaxListBuilder(Of GreenNode) = SyntaxListBuilder(Of GreenNode).Create()
```

Comment:
The added ReadOnly here generated compile error on the following line:

```
_triviaListBuilder(_triviaListBuilder.Count - 1) = _triviaListBuilder(_triviaListBuilder.Count - 1).WithAdditionalDiagnostics(_diagnosticsToAdd.ToArray())
```

> src\Compilers\VisualBasic\Portable\Syntax\InternalSyntax\SyntaxNodeExtensions.vb(542,25): error BC30068: Expression is a value and therefore cannot be the target of an assignment.

I can't see why ReadOnly causes this error??
That line is accessing an indexer on SyntaxListBuilder, how does this violate ReadOnly?

Reply:
Interesting. We'll want a bug filed for this one too.